### PR TITLE
Pepper peru 5

### PR DESCRIPTION
**Pull request by pepper419**
-introduction of bop for peru between army and navy

-reform of the economic tree of Peru

-rebalancing of the equator tree from the war of '41

-rebalancing of some ideas from Peru

### CODE REVIEW

**Review by marijn211 — COMMENTED** [review]
I still think you should remove division attack/defense from the BoP, maybe add some recovery rate instead (maybe (-)15 or 20% at most)
With naval too, would remove naval speed, hit chance and naval damage modifiers, not sure what it should be replaced with there, but your shells arent doing less damage because of morale or prestige

**Review by marijn211 — CHANGES_REQUESTED** [review]
See above

File: history/states/1052-Acre.txt
Comment (ilkober) [inline history/states/1052-Acre.txt:1]: Places 30 rubber instead of 0

File: localisation/english/replace/r56_formables_l_english.yml
Comment (ilkober) [inline localisation/english/replace/r56_formables_l_english.yml:1]: Unifying Hispanoamerica as any other country buy Peru will make you Peru? Why?

**Review by ilkober — COMMENTED** [review]
You added 30 rubber to Acre state???
0 -> 30

**Comment by marijn211** [comment]
Pepper as long as this is marked draft I wont check this, just a reminder

**Comment by marijn211** [comment]
Error log

**Comment by marijn211** [comment]
Previous feedback also still applies, theres still combat stats in the BoP

**Pull request closed without merging**